tracks/python/exercises/reverse-string/mentoring.md

```diff
@@ -0,0 +1,74 @@
+### Problem and challenges
+
+The problem requires the student to reverse a given string.
+
+### Reasonable solutions
+
+[Extended Slice Notation][1] is the fastest way to reverse a string in Python
```

Comment:
Be consistent with punctuation; some sentences end with a period, others do not.